docker/test_dockerimage.sh

```diff
@@ -1,7 +1,9 @@
-#!/bin/bash
+#!/usr/bin/env bash
 set -ex
 
-sudo -u nobody `which emcc` --version
+if [ $EUID -eq 0 ]; then
```

Comment:
Note that this check is needed for Podman (which is the default container engine on Fedora 32). Podman provides a Docker-compatible command line front end that can simply alias the Docker cli (`alias docker=podman`).

Without this check, it will fail to perform the tests (probably because Podman is running in rootless mode).
<details>
  <summary>Details</summary>
  
```bash
$ make -C ./docker version=tot test
make: Entering directory '/home/kleisauke/emsdk/docker'
# test as non-root
docker run --rm -u `id -u`:`id -g` -w /emsdk/docker --net=host emscripten/emsdk:tot \
	bash test_dockerimage.sh
++ which emcc
+ sudo -u nobody /emsdk/upstream/emscripten/emcc --version
sudo: unable to resolve host pc-kaw: Name or service not known

We trust you have received the usual lecture from the local System
Administrator. It usually boils down to these three things:

    #1) Respect the privacy of others.
    #2) Think before you type.
    #3) With great power comes great responsibility.

sudo: no tty present and no askpass program specified
make: *** [Makefile:20: test] Error 1
make: Leaving directory '/home/kleisauke/emsdk/docker'
```
</details>